src/Forms/TreeMultiselectField.php

```diff
@@ -121,7 +121,7 @@ public function getSchemaStateDefaults()
      */
     public function getItems()
     {
-        $value = $this->Value();
+        $value = $this->getValue();
```

Comment:
This is called by both `Field()` and `getSchemaStateDefaults()` - but it's also called by `dataValue()`, so hard to say whether `getValue()` or `getFormattedValue()` is correct - or even if it should be different depending on what's calling this method.
Or maybe the logic in the method requires the raw value?

All this to say - please explain why we're using the raw value and not the formatted value here.

Reply:
There's logic in this method that checks that `$value` is either `$value === 'unchanged'`, `$value instanceof SS_List or `// Parse ids from value string / array`

In all of those cases the raw value seems like it's more appropriate